.github/workflows/CI.yml

```diff
@@ -210,3 +210,26 @@ jobs:
 
       - name: cargo doc
         run: cargo rustdoc --features full,ffi -- --cfg docsrs --cfg hyper_unstable_ffi -D broken-intra-doc-links
+
+  check-external-types:
```

Comment:
Oh I forgot, to make this required, you'd need to add it to the list under `ci-pass.needs`.

Reply:
Good point, added it there.